designsafe/static/scripts/projects/components/manage-field-recon/collections/manage-field-recon-collections.component.js

```diff
@@ -290,18 +288,36 @@ class ManageFieldReconCollectionsCtrl {
         this.data.busy = true;
         let collection = {
             title: this.form.title,
-            observationTypes: this.form.observationTypes,
+            observationTypes: [],
             dateStart: this.form.dateStart,
             dateEnd: this.form.dateEnd,
             dataCollectors: this.form.dataCollectors,
             location: this.form.location,
             longitude: this.form.longitude,
             latitude: this.form.latitude,
             elevation: this.form.elevation,
-            instruments: this.form.instruments,
-            referencedDatas: this.form.referencedDatas,
+            instruments: [],
+            referencedDatas: [],
             description: this.form.description,
         };
+        this.form.observationTypes.forEach((obs) => {
```

Comment:
Instead of creating an empty array and then populating it, could you do something like `observationTypes = this.form.observationTypes.filter(...)`?

Reply:
Ah yeah that would look cleaner.